.pre-commit-config.yaml

```diff
@@ -17,8 +17,7 @@ repos:
       - id: markdownlint
 
   - repo: git://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.12.0
+    rev: v1.17.0
     hooks:
       - id: terraform_docs
       - id: terraform_fmt
-      - id: terraform_validate_no_variables
```

Comment:
why did we remove terraform validate?

Reply:
I think it doesn't work with TF12 yet, if I recall...I can try poking it.

Reply:
I think it was removed in v1.15 and replaced with something else. Also the `terraform_docs` doesn't work right if you happen to not be using gnu-awk.